packages/kit/test/apps/basics/test/server.test.js

```diff
@@ -272,15 +272,18 @@ test.describe('Routing', () => {
 });
 
 test.describe('Shadowed pages', () => {
-	test('Action can return undefined', async ({ request }) => {
+	// TODO this doesn't work anymore because we call formData on the user's behalf,
+	// and posting FORM data will trigger the CSFR 403, which also check for in other tests in this app
```

Comment:
i don't totally understand this, so i'm leaving a comment as a note to self to dig into it

```suggestion
	// TODO this doesn't work anymore because we call formData on the user's behalf,
	// and posting FORM data will trigger the CSRF 403, which also check for in other tests in this app
```